### PR TITLE
Update introduction.md

### PR DESCRIPTION
**Pull request by poshrolls**
please.

### CODE REVIEW

**Comment by pdcastro** [comment]
@poshrolls, VersionBot detected a problem with the commit message: it is missing the `Change-type: patch` footer.
* Docs on amending a commit message:  
https://docs.github.com/en/github/committing-changes-to-your-project/changing-a-commit-message
* Docs on what VersionBot expects:  
https://github.com/balena-io-modules/resin-commit-lint#examples
* Docs on the meaning of patch / minor / major:  
https://semver.org/

Basically, instead of:
```
Update introduction.md
```
We need at least:
```
Update introduction.md

Change-type: patch
```

**Review by vipulgupta2048 — APPROVED** [review]
@poshrolls Real typo found, but we couldn't find any issues linked. So no coins for you :tada: 
And, update what @pdcastro said ^

**Comment by LucianBuzzo** [comment]
Closing in favor of https://github.com/balena-io/docs/pull/1697
